.github/workflows/label-notify.yml

```
@@ -11,7 +11,8 @@ jobs:
         - uses: jenschelkopf/issue-label-notification-action@f7d2363e5efa18b8aeea671ca8093e183ae8f218 # 1.3
           with:
              recipients: |
-                  team/web=@joelkw @felixfbecker @jeanduplessis
+                  team/extensibility=@joelkw @jeanduplessis
+                  team/frontend-platform=@jeanduplessis @alicjasuska
```

I think it would still be benifical to have me receive email notifications for a while in case there is something I have context to share on (but I won't reply usually)
```suggestion
                  team/extensibility=@joelkw @jeanduplessis @felixfbecker
                  team/frontend-platform=@jeanduplessis @alicjasuska @felixfbecker
```